swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SettingsView.swift

```diff
@@ -590,17 +595,23 @@ public struct SettingsView: View {
 
         Task {
           do {
-            try await compressor.compressFolder(destinationURL: destinationURL)
-            self.isExportingLogs = false
+            try await LogExporter.export(to: destinationURL)
+
             await MainActor.run {
               window.contentViewController?.presentingViewController?.dismiss(self)
             }
           } catch {
-            self.isExportingLogs = false
+            Log.app.error("\(#function): \(error)")
```

Comment:
We now show an error to the user if log export on macOS fails, so at least we'll have some signal if this becomes a widespread issue.

Reply:
When Sentry-Swift? :)